packages/webapp/src/components/ResourceList/index.jsx

```diff
@@ -14,7 +14,6 @@ import { Typography } from '@material-ui/core';
 const Title = styled(Typography)`
   && {
     height: 39px;
-    font-family: Lato;
```

Comment:
Confirm with Matt that we want to move away from Lato to Roboto?

Reply:
Checked with Matt and made some further styling minor edits.